BridgeAppSDK/SBAScheduledActivityManager.swift

```diff
@@ -319,6 +319,16 @@ open class SBABaseScheduledActivityManager: NSObject, ORKTaskViewControllerDeleg
      @return                            The filtered list of activities
      */
     open func filteredSchedules(scheduledActivities: [SBBScheduledActivity]) -> [SBBScheduledActivity] {
+        if _loadingState == .fromServerWithFutureOnly {
+            // The future only will be in a state where we already have the cached data,
+            // And we will probably just be appending new data onto the cached data
+            let filteredActivities = scheduledActivities.filter({ (activity) in
```

Comment:
I thought it was the other way around, it would pull future stuff first in case it *didn't* have cached data, and then go backfill the cache in the background? Or is this a different case and .fromServerWithFutureOnly isn't how that scenario is handled?